test/EmailAddressTest.php

```diff
@@ -741,8 +741,8 @@ public function testUseMxCheckBasicValid()
         ];
 
         if (extension_loaded('intl')) {
-            $emailAddresses[] = 'иван@письмо.рф';
-            $emailAddresses[] = 'xn--@-7sbfxdyelgv5j.xn--p1ai';
+            $emailAddresses[] = 'test@письмо.рф';
+            $emailAddresses[] = 'test@xn--h1aigbl0e.xn--p1ai';
```

Comment:
Why the domain part change here? You say the issue is in the local part, so why change the domain part?